hexrd/ui/import_data_panel.py

```diff
@@ -53,7 +53,6 @@ def setup_connections(self):
         self.ui.button_box.rejected.connect(self.clear)
         self.ui.save.clicked.connect(self.save_file)
         self.ui.complete.clicked.connect(self.completed)
-        self.new_config_loaded.connect(HexrdConfig().instrument_config_loaded)
```

Comment:
This had the same effects as https://github.com/HEXRD/hexrdgui/pull/786/files#r588765330

Hopefully, removing it won't have any side effects.